### DISCUSSION TITLE
유저의 화면에 접근하는법

### DISCUSSION THREAD

**Discussion by JeongYire** [naverz/zepeto-studio-kor#279] [5. Q&A]
화면에 data storage 를 연동해 데이터를 계속 보게 만든다던지,
특정 유저의 데이터를 그 유저의 화면에만 띄우게 하고 싶은데

그럴려면 유저의 화면에 접근할 수 있어야합니다.
이 경우 어떻게 해야 할까요? 현재 multiplay sample 을 쓰고있으며 기본 제공되고있는 컨트롤러를 쓰고 있습니다

**Comment by runasy-koonta** [comment]
UI Layer에 접근하시려는 것 같은데 Canvas에 UI 쓰면 되지 않을까요?
https://studio.zepeto.me/guides/ui-events

**Reply by JeongYire** [reply]
아뇨 그게 아니라, 유저가 갖고있는 데이터를 조회해서 UI에 띄울 수 있는 방법이 따로 없나 해서요.
ZEPETOPlayer 의 경우에는 ZEPETOPlayers.instance.getPlayer(sessionId) 를 통해 얻을 수 있다고해도
이건 어디까지나 플레이어 캐릭터 오브젝트의 경우이지, 플레이어 캐릭터에게 보이는 UI 오브젝트를 뜻하지 않으니까요 
현재 방법은 UI 단에다가 RoomBase 얻게해서, 캐릭터 생성시마다 메세지를 보내는 형태로 데이터를 얻고 있긴한데, 왠지 이렇게하면 안될거같아 문의 드려봅니다

**Reply by x4cf** [reply]
의도한대로 작동한다면 그것도 방법이라 생각합니다
